### PR TITLE
Right-Top-Nav - remove multiple ids from breadcrumb. set ids and classes where missing.

### PR DESCRIPTION
**Pull request by clouless**
**TL;DR Minor UI fixes on ids and classnames in breadcrumbBar**

Also see previous PR #2989.

image

### Proposed changelog entries

* breadcrumbBar
  * remove multiple occurrences of `right-top-nav` id from breadcrumb. (IDs should be unique!)
  * set ids and classes for autoRefresh links

### Submitter checklist

- ~[ ] JIRA issue is well described~ 
  * *Minor improvement, so no JIRA issue.*
- [x] Changelog entry appropriate for the audience affected by the change:
  * Internal, developer
- [x] Appropriate autotests or explanation to why this change has no tests
  * I has not tests, since changes a minor UI changes that would require an E2E test.
  * Changes have been tested in multiple browsers.
- ~[ ] For dependency updates: links to external changelogs and, if possible, full diffs~

### Desired reviewers

@i386 as mentioned in PR #2989 I am on small mission to fix UI bugs regarding ids and classes.

### CODE REVIEW

**Comment by daniel-beck** [comment]
Conflicts with #2979 proposing to remove this abomination. I suggest we first figure out what to do there before considering to merge this.

**Comment by clouless** [comment]
@daniel-beck ok cool. I am fine with #2979 and the auto refresh being removed. And maybe you remove the right-top-nav completely and/or fix the duplicate ID thing there :)

**Comment by daniel-beck** [comment]
@clouless I did, the inner element with the same ID is no more: https://github.com/jenkinsci/jenkins/pull/2979/files#diff-6adb308045b89853b8151bf0c27ee28bL44

**Comment by clouless** [comment]
@daniel-beck perfect. Then we can close this as **won't fix** (Since I have hated the auto refresh as well for a long time ;) )
